docs/sdk/cloud_storage.md

```diff
@@ -3,3 +3,7 @@
 ## Queries
 
 ::: kili.queries.data_integration.__init__.QueriesDataIntegration
+
+## Mutations
+
+:::kili.mutations.data_connection.__init__.MutationsDataConnection
```

Comment:
I think that we could also add a little paragraph to explain what is the difference between what we call a data connection and a data integration.
And to to explain that the integration can only be done in the front currently and link to https://docs.kili-technology.com/docs/adding-assets-to-project#creating-a-remote-storage-integration
also @PierreLeveau in the readMe doc, it is called "Remote Storage"

Reply:
Yes I'm going to add some more info in the doc

Reply:
@kili-technology/ml we rename all "cloud storage" to "remote storage"?

Reply:
not a big deal, we will rename it when the renaming will be effective.